legacy/reqcounter/reqcounter.go

```diff
@@ -104,8 +105,8 @@ const (
 	// requests counters to perfectly line up with the actual GCR quota.
 	// Source: https://cloud.google.com/container-registry/quotas
 	MeasurementWindow time.Duration = time.Minute * 10
-	// timestampFormat specifies the syntax for logging time stamps of request counters.
-	timestampFormat string = "2006-01-02 15:04:05"
+	// TimestampFormat specifies the syntax for logging time stamps of request counters.
```

Comment:
Making the timestamp format for logs global allows a reference of this syntax for #370